packages/pcd-types/src/pcd.ts

```diff
@@ -1,74 +1,165 @@
+/**
+ * All PCDs consist of a "claim", which is the human-interpretable statement
+ * that the PCD is making (i.e. "I am a Zuzalu resident"); and a "proof" attached
+ * to the "claim," which is a cryptographic or mathematical proof of the claim.
+ * All PCDs within this SDK also expose a `prove` and `verify` function, which
```

Comment:
"All PCD packages"?  Want to be clear on the separation of what's in the instance vs. the type/package.

This might be a good place to put some extra text explaining the relationship of a PCD subtype and its PCDPackage, and/or to link to the PCDPackage docs which do so.

Reply:
You're right, we often conflate the concept of a PCD with the concept of its corresponding PCD Package. I wonder what pair of names for these concepts would work better. In the meantime, I am going to be sure be precise in my language, so that at least in the documentation I don't conflate the two.